bitnami/redis/values.yaml

```diff
@@ -1412,6 +1412,59 @@ metrics:
     ##   - myRegistryKeySecretName
     ##
     pullSecrets: []
+  ## Configure extra options for Redis&reg; containers' liveness, readiness & startup probes
+  ## ref: https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes/
+  ## @param metrics.startupProbe.enabled Enable startupProbe on Redis&reg; replicas nodes
+  ## @param metrics.startupProbe.initialDelaySeconds Initial delay seconds for startupProbe
+  ## @param metrics.startupProbe.periodSeconds Period seconds for startupProbe
+  ## @param metrics.startupProbe.timeoutSeconds Timeout seconds for startupProbe
+  ## @param metrics.startupProbe.failureThreshold Failure threshold for startupProbe
+  ## @param metrics.startupProbe.successThreshold Success threshold for startupProbe
+  ##
+  startupProbe:
+    enabled: false
+    initialDelaySeconds: 10
+    periodSeconds: 10
+    timeoutSeconds: 5
+    successThreshold: 1
+    failureThreshold: 5
+  ## @param metrics.livenessProbe.enabled Enable livenessProbe on Redis&reg; replicas nodes
+  ## @param metrics.livenessProbe.initialDelaySeconds Initial delay seconds for livenessProbe
+  ## @param metrics.livenessProbe.periodSeconds Period seconds for livenessProbe
+  ## @param metrics.livenessProbe.timeoutSeconds Timeout seconds for livenessProbe
+  ## @param metrics.livenessProbe.failureThreshold Failure threshold for livenessProbe
+  ## @param metrics.livenessProbe.successThreshold Success threshold for livenessProbe
+  ##
+  livenessProbe:
+    enabled: true
+    initialDelaySeconds: 10
+    periodSeconds: 10
+    timeoutSeconds: 5
+    successThreshold: 1
+    failureThreshold: 5
+  ## @param metrics.readinessProbe.enabled Enable readinessProbe on Redis&reg; replicas nodes
+  ## @param metrics.readinessProbe.initialDelaySeconds Initial delay seconds for readinessProbe
+  ## @param metrics.readinessProbe.periodSeconds Period seconds for readinessProbe
+  ## @param metrics.readinessProbe.timeoutSeconds Timeout seconds for readinessProbe
+  ## @param metrics.readinessProbe.failureThreshold Failure threshold for readinessProbe
+  ## @param metrics.readinessProbe.successThreshold Success threshold for readinessProbe
+  ##
+  readinessProbe:
+    enabled: true
+    initialDelaySeconds: 5
+    periodSeconds: 10
+    timeoutSeconds: 1
+    successThreshold: 1
+    failureThreshold: 3
```

Comment:
Default values for the readiness probe may be too strict.

Its `initialDelaySeconds` is shorter than the livenessProbe's initial delay, and the 1 second timeout may be too short too.

I would suggest, at least, 15 seconds for initialDelaySeconds and 5 seconds timeout.

Reply:
Can you please elaborate why?

The kubernetes default for initialDelaySeconds is "0". So "5s" should already giving the exporter enough time to start and be able to receive traffic.

Imho initialDelaySeconds of the readinessprobe should be independent from the initialDelaySeconds setting of the livenessprobe. 

Livenessprobe would fail after "initialDelaySeconds + (periodSeconds * failureThreshold)" = 60s.
Readinessprobe would fail after: "initialDelaySeconds + (periodSeconds * failureThreshold)" = 35s.

So having the readinessprobe failing before the livenessprobe sounds reasonable to me.

The 1 second timeout is also the k8s default and it should also be enough to render a mostly empty html page, as metrics are not part of it. If thats not possible there is likely already a problem and guess i don't want to send more traffic there.

Reply:
I'm sorry, you are absolutely right. It is the liveness that should be performed after the readiness.

With that, the PR looks good to me, thank you very much for your contribution!